.github/workflows/positron-python-ci.yml

```diff
@@ -117,7 +117,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         # Run the tests on the oldest and most recent versions of Python.
-        python: ['3.8', '3.13']
+        python: ['3.9', '3.13']
```

Comment:
should we change nightly to only 3.9+ as well?

Reply:
Good catch, done in 0453138